projects/admin/src/app/routes/documents-route.ts

```diff
@@ -69,10 +69,7 @@ export class DocumentsRoute extends BaseRoute implements RouteInterface {
               .aggregationFilter(aggregations),
             aggregationsOrder: [
               'document_type',
-              'contribution__fr',
-              'contribution__en',
-              'contribution__de',
-              'contribution__it',
+              'author',
```

Comment:
Is this facet should not be named 'contributor' as in next US we will add corporate contributor.

Reply:
At this time, we need to keep `author` as facet label. We cannot translate `contribution` by `author`.